### PR TITLE
Define variable bridge

### PR DESCRIPTION
**Pull request by blegat**
Extracted from https://github.com/JuliaOpt/MathOptInterface.jl/pull/759

### CODE REVIEW

File: src/Bridges/Constraint/bridge.jl [hunk 1]
```diff
@@ -55,22 +40,36 @@ Return a `Bool` indicating whether the bridges of type `BT` support bridging `F`
...
 
-Return a list of the types of constraints that bridges of type `BT` add for
-bridging an `F`-in-`S` constraints.
-
-    added_constraint_types(BT::Type{<:AbstractBridge})::Bool
-
-Return a list of the types of constraints that bridges of concrete type `BT` add
-for `F`-in-`S` constraints.
+Return a list of the types of constrained variables that bridges of type `BT`
+add for bridging an `F`-in-`S` constraints. This fallbacks to
```
Comment (mlubin) [inline src/Bridges/Constraint/bridge.jl]: s/an//

Comment (mlubin) [inline src/Bridges/Constraint/bridge.jl]: s/fallbacks/falls back/ (same below)

File: src/Bridges/Constraint/bridge.jl [hunk 2]
```diff
@@ -82,10 +81,21 @@ function can only be called if `MOI.supports_constraint(BT, F, S)` is `true`.
...
-The following returns `SplitIntervalBridge{Float64, MOI.SingleVariable}`:
-```julia
-concrete_bridge_type(SplitIntervalBridge{Float64}, MOI.SingleVariable,
-                                                   MOI.Interval{Float64})
+As a [`MathOptInterface.SingleVariable`](@ref)-in-[`MathOptInterface.Interval`](@ref)
+constraint is bridged into a
+[`MathOptInterface.SingleVariable`](@ref)-in-[`MathOptInterface.GreaterThan`](@ref)
+and a
+[`MathOptInterface.SingleVariable`](@ref)-in-[`MathOptInterface.LessThan`](@ref)
+by teh [`SplitIntervalBridge`](@ref),
```
Comment (mlubin) [inline src/Bridges/Constraint/bridge.jl]: s/teh/the/

File: src/Bridges/Variable/bridge.jl [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+variables in [`MathOptInterface.Nonnegatives`](@ref) by the
+[`VectorizeBridge`](@ref):
+```jldoctest
+BT = MOI.Bridges.Variable.VectorizeBridge{Float64}
+S = MOI.GreaterThan{Float64}
+MOI.Bridges.added_constrained_variable_types(BT, S)
+
+# output
+
+[(MOI.Nonnegatives,)]
```
Comment (mlubin) [inline src/Bridges/Variable/bridge.jl]: These examples are very helpful. However, I don't understand how the doctests are passing given that the bridges are not implemented yet.

Comment (blegat) [reply src/Bridges/Variable/bridge.jl]: The function docstring is not included in the doc, it is included in the `bl/variable_bridge` branch

**Pull request merged**